### ISSUE TITLE
Could I use Chatterbot completed trained myself

### ISSUE THREAD

**Issue by Leeck0530** [gunthercox/ChatterBot#2088]
Could I use Chatterbot completed trained by myself?
I want to let it  response the answers only in my training list,

**Comment by AnAnalogGuy** [comment]
Yes, you can create your own set of questions and answers an train chatterbot accordingly.
